app/services/iteration_service.rb

```diff
@@ -0,0 +1,59 @@
+class IterationService
+  DAYS_IN_WEEK = (1.week / 1.day)
```

Comment:
shouldn't this be the days of the sprint of project? does the central allow to config a project to a 2 week sprint?